surrealdb/clients/ws.py

```diff
@@ -78,10 +78,11 @@ async def __aexit__(
         exc_value: Optional[BaseException] = None,
         traceback: Optional[TracebackType] = None,
     ) -> None:
+        """Disconnect the websocket client when exiting the context manager."""
```

Comment:
Should this be multiline?

Reply:
Black's default max line length is 88 characters, so in this case no. This can be configured though if you wanted to look at making it shorter